From 572aea93fabf9dcfedce8c6ac34e017128f01062 Mon Sep 17 00:00:00 2001
From: Sean Martin <seankieran.m@hotmail.com>
Date: Fri, 18 Aug 2023 16:13:15 +0100
Subject: [PATCH] Update Kyle's tag on the core devs page (#232)

# Description
I noticed that Kyle's tag on the core devs page was linking to the right
place, but the display name was wrong. So I've updated that here.
---
 docs/community/team.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/community/team.md b/docs/community/team.md
index 04b5d77e..c4ac67ad 100644
--- a/docs/community/team.md
+++ b/docs/community/team.md
@@ -12,7 +12,7 @@ napari is a consensus-based community project. Anyone with an interest in the pr
 - [Juan Nunez-Iglesias](https://github.com/napari/napari/commits?author=jni) - [@jni](https://github.com/jni)*
 - [Kevin Yamauchi](https://github.com/napari/napari/commits?author=kevinyamauchi) - [@kevinyamauchi](https://github.com/kevinyamauchi)*
 - [Kira Evans](https://github.com/napari/napari/commits?author=kne42) - [@kne42](https://github.com/kne42)
-- [Kyle Harrington](https://github.com/napari/napari/commits?author=kephale) - [@royerloic](https://github.com/kephale)
+- [Kyle Harrington](https://github.com/napari/napari/commits?author=kephale) - [@kephale](https://github.com/kephale)
 - [Loic Royer](https://github.com/napari/napari/commits?author=royerloic) - [@royerloic](https://github.com/royerloic)^
 - [Lorenzo Gaifas](https://github.com/napari/napari/commits?author=brisvag) - [@brisvag](https://github.com/brisvag)
 - [Wouter-Michiel Vierdag](https://github.com/napari/napari/commits?author=melonora) - [@melonora](https://github.com/melonora)
